onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4870,13 +4870,23 @@ def aten_xor(self: TensorType, other: TensorType) -> TensorType:
     raise NotImplementedError()
 
 
-def aten_zeros(size: INT64) -> TensorType:
+def aten_zeros(self, size, dtype: int = -1):
     # zeros(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
 
-    raise NotImplementedError()
+    if dtype == -1:
+        zero = op.Constant(value=0)  # type: ignore[arg-type]
+    else:
+        zero = op.Cast(0, to=dtype)  # type: ignore[arg-type]
+    return op.ConstantOfShape(size, zero)
```

Comment:
aten_ones: https://github.com/microsoft/onnx-script/pull/252/files#diff-e94a882bd9dcdfc75120b0571c970a28d35ec65bf9b8ba02b68a8b442f2853a7R3467

Reply:
As long as it runs I am for anything that’s more simple. Were you able to run the tests? Expand was what Rama suggested when I initially used ConstantOfShape, because the shape is dynamic.